docs/src/_static/js/jquery-migrate-1.1.1.min.js

```diff
@@ -1,3 +1,3 @@
-/*! jQuery Migrate v1.1.1 | (c) 2005, 2013 jQuery Foundation, Inc. and other contributors | jquery.org/license */
+/*! jQuery Migrate v1.1.1 | (c) 2005, 2013 jQuery Foundation, Inc. and other contributors | jquery.org/license */
```

Comment:
can you simply checkout this file, please?

Reply:
:D

Reply:
Sure